src/software/firmware/test/test_pression.cpp

```diff
@@ -0,0 +1,33 @@
+/**
+ * @file test_pression.cpp
+ * @author Makers For Life
+ * @brief Unit tests for pression.cpp
+ * @version 0.1
+ * @date 2020-04-06
+ * 
+ * @copyright Copyright (c) 2020
```

Comment:
In the other source files I had to put the copyright notice on its own comment to make cpplint happy. But if this work it would be nice to port this in the other files! (in another PR of course)

Reply:
The linter seems happy with it